geanydoc/src/geanydoc.c

```diff
@@ -336,7 +332,9 @@ create_Configure(void)
 	gtk_window_set_title(GTK_WINDOW(Configure), _("Doc"));
 	gtk_window_set_type_hint(GTK_WINDOW(Configure), GDK_WINDOW_TYPE_HINT_DIALOG);
 
-	dialog_vbox1 = GTK_DIALOG(Configure)->vbox;
+	dialog_vbox1 = gtk_vbox_new(FALSE, 6);
```

Comment:
also, you could assume just fine that `gtk_dialog_get_content_area()` returns a VBox if you wanted